packages/bluefin/bluefin.spec

```diff
@@ -124,7 +126,9 @@ Contains all of the DConf settings that Bluefin ships by default
 %{_datadir}/glib-2.0
 %{_datadir}/applications
 %{_datadir}/ublue-os/homebrew/*.Brewfile
+%if (0%{?fedora} && 0%{?fedora} < 43)
```

Comment:
Sorry, could you explain what this condition means?  
Bluefin also runs on CentOS Stream 10, so would this still be present there?